api.go

```diff
@@ -87,6 +94,8 @@ type LineFragmentMatch struct {
 
 	// Number bytes that match.
 	MatchLength int
+
+	SymbolInfo *Symbol
```

Comment:
needs a docstring. Also I am assuming this is nil if it is not from a  symbol search? You should say that.